test/integration/targets/docker_image/tasks/tests/options.yml

```diff
@@ -0,0 +1,30 @@
+####################################################################
+## load path #######################################################
+####################################################################
+
+- name: Archive image
+  docker_image:
+    name: "hello-world:latest"
+    archive_path: image.tar
+
+- name: remove image
+  docker_image:
+    name: "hello-world:latest"
+    state: absent
+
+- name: load image (changed)
+  docker_image:
+    name: "hello-world:latest"
+    load_path: image.tar
+  register: load_image_1
```

Comment:
What you could also do is get the image ID (from the first step, if it returns it, otherwise use `docker_image_facts`) and compare it to the loaded image's ID (which should of course be the same).